source/develop/release-management/features/network/managed_mtu_for_vm_networks.html.md

```diff
@@ -191,6 +193,14 @@ version of [OpenStack Java SDK][11] used by oVirt-engine.
   build 139) or later, which is already given for the already released
   oVirt-toolsSetup 4.2-1.
 
+* MTU cannot applied to pass trough vNICs.
+
+* The attributes link state, device type of the vNIC, and the attributes port
```

Comment:
"Currently the attributes `link state`, `device type` of the vNIC, port mirroring, custom properties, QoS, and the network of the vNIC profile cannot be updated for plugged vNICs. As a workaround, unplug the vNIC, change the attribute, and replug the vNIC."

Reply:
Done